web/src/dialog_widget.ts

```diff
@@ -190,6 +190,7 @@ export function launch(conf: DialogWidgetConfig): void {
 
     // Set up handlers.
     $submit_button.on("click", (e) => {
+        e.preventDefault();
```

Comment:
I'm wondering if we need to add `e.stopPropagation();` here as well.

Reply:
Yeah I think we should add it.
I looked at every user of `dialog_widget` and found that every user is calling both `preventDefault` and `stopPropagation` calls.